entry_types/scrolled/package/spec/support/rendering.js

```diff
@@ -18,7 +18,23 @@ import {normalizeSeed} from './normalizeSeed';
  * The `setup` option can be used to simulate rendering the component
  * in the editor where data is synchronized from Backbone models.
  *
- * @param {React.Component} ui - React component
+ * To be able to render components that expect the result of certain
+ * hooks as part of their props, instead of a React component, you can
+ * pass a function returning a React component as first parameter. The
+ * function will be evaluated in a context of a React component and
+ * can thus make use of hooks
+ *
+ *     // DOES NOT WORK
+ *     renderInEntry(<Image file={useFile({collectionName: 'imageFiles', permaId: 4})} />, {seed});
```

Comment:
**praise:** I think this is an error that is likely to happen for those who don't use `renderInEntry` frequently. Having 'does not work' and 'works' directly adjacent makes it easy to move on in this kind of situation